RomM.csproj

```diff
@@ -71,7 +71,7 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
iirc the version here _needs_ to match what's shipped with playnite, even if the built-in ones are out of date

Reply:
Are you sure cause locally when i run this Plugin in development mode it works as expected even with this upgrade 👀

Reply:
yes pretty sure, iirc it'll work in dev but fail when installed via the plugin library. i'm referencing the version from this file: https://github.com/JosefNemec/Playnite/blob/5dfb0b1dd9b407fded564a258c42232bd19f70cf/source/Playnite/packages.config#L19

will keeping it at v10 mean you'll need to make other changes in this PR?

Reply:
No, i'll only need to downgrade the dependency 😄